tests/unit/expire.tcl

```diff
@@ -882,17 +880,12 @@ start_cluster 1 0 {tags {"expire external:skip cluster"}} {
             fail "bgsave did not stop in time."
         }
 
-        # Verify dict is under rehashing
-        set htstats [r debug HTSTATS 0]
-        assert_match {*rehashing target*} $htstats
-
```

Comment:
so we're not validating that the rehashing started?
i suppose it would be a race condition if it ends before we detected that it started.
but maybe we can at least validate that rehashing was indeed performed, by verifying that the oattern we match we have at the end of the test, isn't matched earlier (while the fork still exists)?

Reply:
Yeah, it was leading to a race. I could verify the size of HT at this point is 128 and later it reduces to 4. Does that work ?